### PR TITLE
Jetpack Pricing | Add amount saved in bundles to featured item card

### PR DESCRIPTION
**Pull request by manzoorwanijk**
#### Proposed Changes

* This PR adds amount saved text to featured item cards for bundles on Jetpack pricing page.

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-pricing/featured/amount-saved`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Go to Bundles tab
- Confirm that you see "Save {{amount}}/mo vs buying individually" text inside featured card
- Hover over it
- Confirm that you see a tooltip that says, `For details, click on "More about {{product}}"`
- Confirm that the design matches figma

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Completes: 0-as-1203463248826787/f

Screenshot 2022-12-06 at 10 49 19 AM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203463248826787

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx [hunk 1]
```diff
@@ -33,6 +36,11 @@ export const MostPopular: React.FC< MostPopularProps > = ( {
 		getOnClickPurchase,
 		isMultisite,
 	} = useStoreItemInfoContext();
+	const translate = useTranslate();
+
+	const isEnglish = config< Array< string | undefined > >( 'english_locales' ).includes(
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx:41]: I believe you can add the label `String Freeze` to your PR to ensure new strings are translated before merging.

**Review by monsieur-z — CHANGES_REQUESTED** [review]
It works as expected and code looks good.

However, the added element doesn't look clickable. That's an accessibility failure.

**Comment by jeffgolenski** [comment]
Functions well @manzoorwanijk! - @monsieur-z what would you recommend there? I personally thought the info icon covers that ground.

File: client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx (hunk 1 above)
Comment (manzoorwanijk) [reply client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx:41]: Added. Thanks

**Comment by manzoorwanijk** [comment]
> However, the added element doesn't look clickable. That's an accessibility failure.

Thank you for catching that @monsieur-z. How can we fix that?

CC: @ilonagl

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7651297

Thank you @manzoorwanijk for including a screenshot in the description! This is really helpful for our translators.

**Comment by ilonagl** [comment]
I'd echo Jeff's thought that IMO the info icon covers that ground.
Since the same lightboxes can be opened with the links below ("More about.."), this can be seen as an optional link.
Any thoughts?

**Comment by monsieur-z** [comment]
> I personally thought the info icon covers that ground.

I didn't realize the link was clickable until I read the PR complete instructions. My impression is that it is drowned in the surrounding text, which has a very similar colour. 

> Since the same lightboxes can be opened with the links below ("More about.."), this can be seen as an optional link. Any thoughts?

My suggestion is if it does not look clickable, let's not make it clickable—especially since clicking _More about X_ below triggers the same behaviour. If we want it clickable, I'd say that the combination of the icon and a contrasting colour is sufficient.

**Comment by ilonagl** [comment]
Thank you for your reply, @monsieur-z! 

**One option** is to make the link green + underline (like other links in Jetpack.com) as in the image below. 

Pros: it's easier to understand that it's clickable. Cons: it fights for attention with the link "More about.." below. Possible solution: start tracking which link people click more and adapt designs based on that. However, this might take a while until we get tangible results.  

Bundles view-4

**Another option** is to not make it clickable. 

Pros: no worries about making it look clickable nor fight for attention with a link that opens up below. Cons: I worry that if we present a saving information without immediate related link with more info, we might hurt conversions. 

Any thoughts? Maybe there is another solution I'm not aware of? cc @monsieur-z @jeffgolenski

**Comment by ilonagl** [comment]
**Note:** The new lightbox design is already in production, but it's no way to know about savings unless a user clicks on "More about...". I'd suggest moving with what we have and having a separate ticket/task to figure this out. I'm hesitant to keep the task only partly released due to this discussion.

**Comment by manzoorwanijk** [comment]
@monsieur-z @ilonagl @jeffgolenski, what about showing a tooltip on that text and info icon that reads something like, `For details, click on "More about Security"`?

**Comment by ilonagl** [comment]
@manzoorwanijk yes, I think that would do the work. Do you need any designs for this, or is it straightforward enough?

**Comment by monsieur-z** [comment]
> I'd suggest moving with what we have and having a separate ticket/task to figure this out.

Sure, let's handle this separately.

> Any thoughts?

I like your first option (with the green underlined link) best, @ilonagl. I'm personally not too worried that it competes with the _More about X_ link, since they have the same behaviour.

**Comment by manzoorwanijk** [comment]
@ilonagl @monsieur-z I have updated the PR to use the tooltip. I have thus updated the PR description along with the screenshot. Please have another look.

**Review by monsieur-z — APPROVED** [review]
I'm not fond of the tooltip approach, but code looks good.

**Review by ilonagl — APPROVED** [review]

**Comment by eeeeevon13** [comment]
I'd suggest you either remove the icon or create a proper tool tip that links to the detail popover .
See my the video as to how info icons behave elsewhere. We should also make it on click not on hover
[instead.webm](https://user-images.githubusercontent.com/49159284/206474090-2bb83439-df3c-4a96-8fc9-3f30b9877fce.webm)

**Comment by ilonagl** [comment]
Proposed designs using popover: 
Bundles view-5

https://user-images.githubusercontent.com/100145243/206673800-ee1fcebd-753a-49dc-8732-9f5e09df73a6.mov

Figma prototype: https://www.figma.com/proto/m0y1jHTuDpvMPkFsOJBoNP/Pricing-page-to-drive-more-single-plugin-sales?node-id=5089%3A158892&scaling=min-zoom&page-id=4927%3A147819&starting-point-node-id=5089%3A156835&hide-ui=1

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7651297).

**Pull request merged**